challenges-services/src/main/java/org/exoplatform/challenges/listener/AnnouncementActivityGenerator.java

```diff
@@ -62,9 +71,9 @@ private ExoSocialActivity createActivity(Announcement announcement) throws Objec
     params.put("announcementAssigneeUsername", this.getAssigneeUserNames(announcement.getAssignee()));
     params.put("announcementAssigneeFullName", this.getAssigneeFullNames(announcement.getAssignee()));
```

Comment:
If a user changes his fullName the activity won't be correct, we need to retrieve the fullName from Front UI.
No need to save the full name in the activity